pkgs/top-level/python-packages.nix

```diff
@@ -18367,8 +18367,6 @@ self: super: with self; {
 
   unasync = callPackage ../development/python-modules/unasync { };
 
-  unblob-native = callPackage ../development/python-modules/unblob-native { };
```

Comment:
please add unblob-native to pkgs/top-level/python-aliases.nix

Reply:
Added